runner.yml

```diff
@@ -84,7 +88,7 @@ Parameters:
     Type: 'String'
     Description: |
       Timezone in which non-working hours are evaluated.
-    Default: 'Europe/Rome'
+    Default: 'Europe/Paris'
```

Comment:
Not a very relevant change, but I'd stick to the previous default. 😄 🇮🇹 🇫🇷 Or maybe we should use UTC. It's still configurable, after all.